app/src/main/java/kr/ac/konkuk/gdsc/gdscsuyeon/data/TodoDao.kt

```diff
@@ -0,0 +1,26 @@
+package kr.ac.konkuk.gdsc.gdscsuyeon.data
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.Update
+
+@Dao
+interface TodoDao {
+    @Query(value = "SELECT * FROM Todos")
+    fun getAllTodo(): List<Todo>
```

Comment:
동기적으로 List를 받아오기에 suspend fun으로 정의하시거나 Flow 형태로 받아오시는 것을 권장합니다.

```suggestion
    @Query(value = "SELECT * FROM Todos")
    suspend fun getAllTodo(): List<Todo>
```

```suggestion
    @Query(value = "SELECT * FROM Todos")
    fun getAllTodo(): Flow<List<Todo>>
```